README.md

```diff
@@ -365,6 +365,27 @@ This allows you to have just one build artifact that you can use in different en
 
 **Tip:** Hops ships with integrated support for [`dotenv`](https://github.com/motdotla/dotenv), which means that it will try to read a `.env` file from your application root directory and load its values as environment variables.
 
+#### Exposing values to the client
+
+Hops requires you to whitelist specific values from your configuration that will be exposed to the client side. In order to configure which values you need access to on the client side, you can map the configuration keys to boolean values. For nested objects you can use the dot notation:
```

Comment:
```suggestion
Hops requires you to whitelist all values from your configuration that needs to be exposed to the client side. In order to do so, you can map the configuration keys to boolean values. For nested objects you can use the dot notation:
```

We could also write `accessible on the client` instead of `exposed to the client`.